spec/eth/ens/resolver_spec.rb

```diff
@@ -0,0 +1,42 @@
+require "spec_helper"
+
+describe Ens::Resolver do
+
+  let(:geth_dev_http_path) { "http://127.0.0.1:8545" }
+  let(:geth_dev_http) { Client.create geth_dev_http_path }
+  let(:resolver) { Ens::Resolver.new(geth_dev_http) }
+
+  describe ".normalize" do
+    it "can normalize ascii" do
+      expect(resolver.normalize('foo.eth')).to eq('foo.eth')
+    end
+
+    it "can normalize non-ascii (fold)" do
+      # with cyrillic 'o'
+      expect(resolver.normalize('fоо.eth')).to eq('foo.eth')
+    end
+
+    it "can normalize caps" do
+      expect(resolver.normalize('Foo.eth')).to eq('foo.eth')
+    end
+
+    it "can normalize emoji domains" do
+      expect(resolver.normalize('🦚.eth')).to eq('🦚.eth')
+    end
+  end
+
+  describe ".namehash" do
```

Comment:
Nit: you're describing an instance method, so should use `#` instead of `.`. Same for line 9 and 36.

```suggestion
  describe "#namehash" do
```